### PR TITLE
Log in

### PR DESCRIPTION
**Pull request by LouOfficial**

### CODE REVIEW

File: ResidenceManager/ChatMessage.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import Firebase
```
Comment (ginpei) [inline ResidenceManager/ChatMessage.swift]: It looks this line affects nothing. Can we remove it and same lines in the other files?

File: ResidenceManager/HomeViewController.swift [hunk 2]
```diff
@@ -125,6 +127,15 @@ class HomeViewController: UIViewController {
             vc.poster = poster
         }
     }
+    
+    func logout() {
+        try! Auth.auth().signOut()
+        
+        // TODO declare the id as a constant
```
Comment (ginpei) [inline ResidenceManager/HomeViewController.swift:134]: Please do this TODO task 😃

File: ResidenceManager/HomeViewController.swift [hunk 3]
```diff
@@ -233,16 +244,19 @@ extension HomeViewController: UITableViewDelegate, UICollectionViewDelegate {
...
-            let vc = storyboard?.instantiateViewController(withIdentifier: identifier)
-        {
-            switch identifier {
-            case "profile":
-                (vc as! ProfileViewController).user = User.loginUser
-            default: break
+        if let identifier = menuItem[1] as? String {
+            if identifier == "logout" {
+                logout()
+            }
```
Comment (ginpei) [inline ResidenceManager/HomeViewController.swift:250]: That's right I should've considered this case. Thanks!

File: ResidenceManager/User.swift [hunk 4]
```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import Firebase
 
 class User {
```
Comment (ginpei) [inline ResidenceManager/User.swift:12]: Can you extend `Firebase.User` to create our `User` class so that we can handle it in simple way.

Comment (LouOfficial) [reply ResidenceManager/User.swift:12]: NO!

File: ResidenceManager/ChatMessage.swift (hunk 1 above)
Comment (LouOfficial) [reply ResidenceManager/ChatMessage.swift]: Done

**Comment by ginpei** [comment]
OK let's merge them once and refactor in master branch since they are at least working! 😄

**Pull request merged**